packages/colombe/colombe.0.9.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+license:      "MIT"
+authors:      [ "Romain Calascibetta <romain.calascibetta@gmail.com>" "Charles-Edouard Lecat" ]
+maintainer:   [ "Romain Calascibetta <romain.calascibetta@gmail.com>" "Charles-Edouard Lecat" ]
```

Comment:
This line causes a linting error: maintainer email address missing.

Is there a reason the info is ommitted? We can make an exception if needed, but if it's just missing because it was forgotten then might as well add it.

(Same questions for the other packages.)

Reply:
So actually, I will probably redo this release with the right name and an email address. But the linter should be less stricter when, at least, one email address is given into these fields :) .